meson_scripts/init.py

```diff
@@ -95,8 +95,9 @@ def init_submodules(method = 'auto'):
     submodule_status(alt_name_codi, sha_version_codi)
     submodule_status(alt_name_medi, sha_version_medi)
     submodule_status(alt_name_opdi, sha_version_opdi)
-    submodule_status(alt_name_meson, sha_version_meson)
-    submodule_status(alt_name_ninja, sha_version_ninja)
+    if (os.environ.get('SU2_MESON') != 'no'):
```

Comment:
Who sets this environment variable?

Reply:
> Who sets this environment variable?

The user will manually set this variable, if he/she prefers to use system-wide Meson and Ninja.
As documented in the commit message:

    $ SU2_MESON='no' ./meson.py

and then run system-wide Meson and Ninja.